sycl/include/CL/sycl/accessor.hpp

```diff
@@ -741,11 +741,12 @@ class accessor :
   using reference = DataT &;
   using const_reference = const DataT &;
 
-  template <int Dims = Dimensions>
-  accessor(
-      detail::enable_if_t<Dims == 0 && ((!IsPlaceH && IsHostBuf) ||
-                                (IsPlaceH && (IsGlobalBuf || IsConstantBuf))),
-                  buffer<DataT, 1>> &BufferRef)
+  template <int Dims = Dimensions,
```

Comment:
Did you use clang-format?

Reply:
Yes

Reply:
Looks like clang format can't handle this file. But what I can see if copy-paste this function to empty file and apply clang-format:
```
  1 {
  2   template <int Dims = Dimensions, typename AllocatorT,
  3             typename detail::enable_if_t<
  4                 Dims == 0 && ((!IsPlaceH && IsHostBuf) ||
  5                               (IsPlaceH && (IsGlobalBuf || IsConstantBuf)))> * =
  6                 nullptr>
  7   accessor(buffer<DataT, 1, AllocatorT> & BufferRef)
  8 #ifdef __SYCL_DEVICE_ONLY__
  9       : impl(id<AdjustedDim>(), BufferRef.get_range(), BufferRef.MemRange) {
 10 #else
 11       : AccessorBaseHost(
 12             /*Offset=*/{0, 0, 0},
 13             detail::convertToArrayOfN<3, 1>(BufferRef.get_range()),
 14             detail::convertToArrayOfN<3, 1>(BufferRef.MemRange), AccessMode,
 15             detail::getSyclObjImpl(BufferRef).get(), AdjustedDim,
 16             sizeof(DataT)) {
 17     detail::EventImplPtr Event =
 18         detail::Scheduler::getInstance().addHostAccessor(
 19             AccessorBaseHost::impl.get());
 20     Event->wait(Event);
 21 #endif
 22   }
 23 }
```
Could you please do something like this? Or at least keep lines length less than 80 characters?